DESCRIPTION

```diff
@@ -38,7 +38,8 @@ Suggests:
     testthat,
     MASS,
     sampling,
-    XML
+    XML,
```

Comment:
Ouch. This should be xml2... in a different PR.

Reply:
Just to follow up on this, I don't see that we are using `xml2` anywhere in the package. Are you sure that we should change this? :)